db/migrate/20220729095120_update_is_other_type_of_claim_in_table_claims.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateIsOtherTypeOfClaimInTableClaims < ActiveRecord::Migration[7.0]
+  def up
+    execute("UPDATE claims SET is_other_type_of_claim = true WHERE other_claim_details IS NOT NULL;")
```

Comment:
This will only set them to true where other claim details is not null - the remaining claims will not have it set at all (i.e. the boolean column will still be set to NULL) - but this needs setting to false